lib/csv-safe.rb

```diff
@@ -4,17 +4,17 @@
 # Override << to sanitize incoming rows
 # Override initialize to add a converter that will sanitize fields being read
 class CSVSafe < CSV
-  def initialize(data, options = {})
+  def initialize(data, **options)
     options[:converters] = [] if options[:converters].nil?
     options[:converters] << lambda(&method(:sanitize_field))
-    super
+    super(data, **options)
   end
```

Comment:
Hey @xxx, if you have some free time, could you check on this version of the changes? Just the lines I'm adding a comment to right now is the most critical bit. I noticed your version of the code fix worked differently from how I would have done it, and I'm not sure if it's because you have some ruby knowledge that I don't.